### ISSUE TITLE
Migrate TheCombine targets to Ubuntu 20.04

### ISSUE THREAD

**Issue by jmgrady** [sillsdev/TheCombine#995]
The current deployment of _TheCombine_ is running on Ubuntu 18.04 (bionic beaver).

Update the instructions and current instances to run _TheCombine_ on Ubuntu 20.04 (focal fossa).

**Comment by johnthagen** [comment]
@jmgrady If we drop Ubuntu 18.04 as a development target as well, we could also bump our minimum Python version from 3.6 to 3.8, which could be nice (though not a huge maintenance issue if we want to still keep 18.04 support).

**Comment by jmgrady** [comment]
@johnthagen We also need to support 3.7 - that is the version that is installed in the `certmgr` container (based on `debian:buster`).  The other option would be to build the certmgr off of `ubuntu:20.04`.  Note that `python3` is currently not available in any other container.

**Comment by johnthagen** [comment]
Good to know. We could bump up to 3.7 then if we drop Ubuntu 18.04. That has `dataclasses` natively, which simplifies things a little bit

**Comment by johnthagen** [comment]
Related: #1114

**Comment by johnthagen** [comment]
Because I was curious, I went an confirmed the `certmgr` container:

```
$ docker-compose exec certmgr bash
root@6793c76e1006:/opt/certmgr# python3 --version
Python 3.7.3
```

**Comment by jmgrady** [comment]
Closed in #1177
